### DISCUSSION TITLE
BlenderBot 2.0 Persona

### DISCUSSION THREAD

**Discussion by ebuehrle** [facebookresearch/ParlAI#3880] [Q&A] [answered]
Thanks for this wonderful piece of work! Does blenderbot2 have a preset persona somewhere?

**Comment by klshuster** [comment, marked as answer]
BB2 does not have a set persona, however, if you use `parlai interactive` and specify `--task convai2`, it will provide BB2 with a persona

**Reply by apsyio** [reply]
Hi, could you pls elaborate on this? For example, would it be possible to make BB2 interrogate the other party - to find specific information - rather than provide free-from responses?
